src/objects/core/zcl_abapgit_objects_activation.clas.abap

```diff
@@ -196,7 +198,21 @@ CLASS zcl_abapgit_objects_activation IMPLEMENTATION.
 
     ELSE.
 
-      activate_old( ii_log ).
+      " Activate transformations first
```

Comment:
this object-type logic is what we want to avoid.

Reply:
Is, adding a new activation step 1.5-XSLT between 1-DDIC and 2-ABAP, fine?

Reply:
Should I propose a commit to have a new activation step 1.5-XSLT between 1-DDIC and 2-ABAP, or any other solution you'd prefer?